app/src/main/java/com/shoots/shoots_ui/ui/payouts/PayoutsFragment.kt

```diff
@@ -0,0 +1,247 @@
+            ) {
+                Text(
+                    "Losers",
+                    style = MaterialTheme.typography.headlineSmall,
+                    color = colorResource(R.color.dark_red)
+                )
+                Text(
+                    "pay the stake to each winner!",
+                    style = MaterialTheme.typography.bodyLarge
+                )
+            }
+        }
+        items(losers) { ranking ->
+            val payout = payouts[ranking.user.id] ?: 0.0
+            PayoutItem(ranking, payout, navController, isWinner = false)
+        }
+    }
+}
+
+@OptIn(ExperimentalGlideComposeApi::class)
+@Composable
+fun PayoutItem(rank: Ranking, payout: Double, navController: NavController, isWinner: Boolean) {
+    val cardColor =
+        if (isWinner) colorResource(R.color.light_green) else colorResource(R.color.light_red)
+
+    Card(
+        modifier = Modifier.padding(vertical = 4.dp),
+        colors = CardDefaults.cardColors(containerColor = cardColor)
+    ) {
+        Row(
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Row(
+                modifier = Modifier.weight(1f),
+                horizontalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                Text("#${rank.rank}", style = MaterialTheme.typography.bodyLarge)
+                Text(
+                    rank.user.name,
+                    style = MaterialTheme.typography.bodyLarge,
+                    fontWeight = FontWeight.Bold
+                )
+            }
+            if (isWinner) {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.spacedBy(8.dp)
+                ) {
+                    GlideImage(
+                        model = "https://cdn-icons-png.flaticon.com/512/5501/5501360.png",
+                        contentDescription = "Earnings",
+                        modifier = Modifier
+                            .size(32.dp)
+                            .clip(CircleShape),
+                    )
+                    Text(
+                        formatUSD(payout),
+                        style = MaterialTheme.typography.bodyLarge,
+                        color = colorResource(R.color.dark_green)
+                    )
+                }
+            } else {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.spacedBy(8.dp)
+                ) {
+                    GlideImage(
+                        model = "https://media.istockphoto.com/id/959957308/vector/money-loss-vector-illustration-flat-cartoon-cash-with-down-arrow-stocks-graph-concept-of.jpg?s=612x612&w=0&k=20&c=HQLE9KYuUiXfHiuYZX7MTVPviDwY96g72gCBWZXsfhg%3D",
```

Comment:
Nit: we should prob move this to a constant, not sure if he'll deduct points for this